config/paths.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
   backendEntryPoints: {
     sourcecred: resolveApp("src/cli/main.js"),
     //
```

Comment:
I know it's not added in this commit, but: is there any reason for the empty comment here? If not, maybe we can  fly-by remove it? (approved either way)

Reply:
When we had an OClif CLI, it separated the OClif commands from the other
various entry points. I’m fine with removing it.